server/controllers/amp-page.js

```diff
@@ -71,6 +71,7 @@ function getAndRender(uuid, options) {
 
 			const shareParams = {};
 			article.shareUrl = `${article.webUrl}?${querystring.stringify(shareParams)}`;
+			article.facebookAppId = '121355018262104';
```

Comment:
Um. I think it's probably fine? I'd rather not have to set up an entirely new app, and it doesn't affect anything to do with the sharing dialog.